urdf_parser_plugin/package.xml

```diff
@@ -16,8 +16,13 @@
   <url type="repository">https://github.com/ros2/urdf</url>
   <url type="bugtracker">https://github.com/ros2/urdf/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_export_depend>ament_cmake_ros</buildtool_export_depend>
   <build_depend>liburdfdom-headers-dev</build_depend>
-  <run_depend>liburdfdom-headers-dev</run_depend>
+  <exec_depend>liburdfdom-headers-dev</exec_depend>
```

Comment:
Hm, it feels a bit weird to have an `exec_depend` on a development library, particularly one that is header-only.  Is this necessary?

Reply:
Opps, `<build_export_depend>` in f9b2176